generator/send_receive.py

```diff
@@ -0,0 +1,54 @@
+import sys
+sys.path.append("ParseCAN")
+import ParseCAN
+from common import send_recieve_path, coord, templ, ifndef, endif, frame_handler
+
+
+def define_pub_frame(frame, name_prepends, busnm, fw):
+    tot_name = coord(name_prepends, frame.name,
+        prefix=False)
+    fw('void send_{}_msg(CANlib_{}_T *inp)'.format(
+        tot_name, tot_name) + ' {\n'
+    )
+
+    if frame.period is not None:
+        fw('\tLIMIT(CANlib_{});\n'.format(tot_name))
+
+    fw(
+        '\tFrame frame;\n' +
+        '\tCANlib_Pack_' + tot_name + '(inp, &frame);\n' +
+        '\tCANlib_TransmitFrame(&frame,' + busnm + ');\n'
+        '}\n\n'
+    )
+
+
+def define_sub_frame(frame, name_prepends, fw):
+    tot_name = coord(name_prepends, frame.name,
+        prefix=False)
+    fw('void handle_{}_msg(Frame *frame)'.format(
+        tot_name, tot_name) + ' {\n' + '\tCANlib_Unpack_{}(frame, &CANlib_{}_inp);\n'.format(tot_name, tot_name) + '}\n\n')
+
+
+def define_struct(frame, name_prepends, fw):
+    tot_name = coord(name_prepends, frame.name)
+    fw('{}_T {}_inp;\n'.format(
```

Comment:
I'd prefer something to the effect of `CANlib_Storage_<name>`.

Reply:
I can change it to have a more conventional name but I like the input part at the end--what about `CANlib_<name>_Input` ?